src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/component/LastKnownAddressComponentTest.kt

```diff
@@ -109,9 +111,33 @@ class LastKnownAddressComponentTest : ComponentTestBase() {
     )
 
     val expectedStatus = HttpStatus.NOT_FOUND
-    val response = authenticatedClient.lastKnownAddress(createLastKnownAddressRequest, expectedStatus, ErrorResponse::class.java)
+    val response = authenticatedClient.addLastKnownAddress(notFoundRecallId, createLastKnownAddressRequest, expectedStatus, ErrorResponse::class.java)
 
     assertThat(response, present())
     assertThat(response, equalTo(ErrorResponse(expectedStatus, "RecallNotFoundException(recallId=$notFoundRecallId)")))
   }
+
+  @Test
+  @Transactional
```

Comment:
We're not using `@Transactional` on any other component tests that I can see ... so why here?